public/app/features/datasources/state/actions.ts

```diff
@@ -5,7 +5,7 @@ import { updateNavIndex } from 'app/core/actions';
 import { getBackendSrv } from 'app/core/services/backend_srv';
 import { getDatasourceSrv } from 'app/features/plugins/datasource_srv';
 import { importDataSourcePlugin } from 'app/features/plugins/plugin_loader';
-import { getPluginSettings } from 'app/features/plugins/PluginSettingsCache';
+import { getPluginSettings } from 'app/features/plugins/components/PluginSettingsCache';
```

Comment:
nit: is this a component?

Reply:
You are right, it's not 🤦‍♂️ 

Probably the name deceived me, haha.

Reply:
Fixed in https://github.com/grafana/grafana/pull/41686/commits/767b16f237cdbc308a6ee166da312180f98a8c18.